addons/web/static/tests/qunit.js

```diff
@@ -286,13 +286,18 @@ export function setupQUnit() {
             console.log("test successful");
         } else {
             console.error(errorMessages.join("\n"));
+            errorMessages.length = 0;
         }
     });
 
     /**
      * This is done mostly for the .txt log file generated by the runbot.
      */
     QUnit.moduleDone((result) => {
+        if (errorMessages.length > 0) {
+            console.error(errorMessages.join("\n"));
+            errorMessages.length = 0;
```

Comment:
what do you want to achieve here? Can't you just empty the array?

Reply:
same problem as the previous comment